pom.xml

```diff
@@ -501,23 +504,44 @@
         </plugins>
       </build>
 
+
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>31.1-jre</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+
       <dependencies>
+        <dependency>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
```

Comment:
scope test missing

Reply:
Getting compilation error in Build when scope test is added, Verified in local. Hence removed the tag.

Reply:
Earlier it was added in dependency management not in the dependency.

Reply:
Earlier added in all dependency including dependency management, but later removed in these as compilation error in build occurred. Added now again to re-check and observed the same error. Hope will need to remove in these dependency. Please confirm so as to remove in L-522 and L-528

Reply:
We can just leave both `sl4j-api` and `guava` in dependency management section and no need to add them as explicit dependency.

Reply:
Was able to add sl4j-api in dependency management and not guava as facing java.io.OutputStream com.google.common.io.ByteStreams.nullOutputStream() in build. Changes done accordingly.